tests/boot/boot_to_desktop.pm

```diff
@@ -26,6 +27,12 @@ sub run {
     else {
         $self->wait_boot(bootloader_time => $timeout);
     }
+    if (is_sle && !check_var('DESKTOP', 'textmode')) {
```

Comment:
Why restrict to SLE? I guess we should apply a common way for both openSUSE and SLE

Reply:
There is the same problem as https://openqa.suse.de/tests/1791801#step/boot_to_desktop/8 in the openSUSE